Daf.Meta.Editor/ViewModels/MainViewModel.cs

```diff
@@ -471,36 +513,44 @@ private void CopyDataSource(Type? windowType)
 
 			if (_windowService.ShowDialog(windowType, vm))
 			{
-				DataSource clonedDataSource = SelectedDataSourceSingle.Clone();
-
-				clonedDataSource.Name = vm.Name;
-				clonedDataSource.SourceSystem = vm.SourceSystem;
-				clonedDataSource.Tenant = vm.Tenant;
-
-				// Replace names with new ones
-				clonedDataSource.FileName = $"{clonedDataSource.Tenant.ShortName}_{clonedDataSource.Name}";
-				clonedDataSource.QualifiedName = $"{clonedDataSource.SourceSystem.ShortName}_{clonedDataSource.FileName}";
-
-				foreach (Satellite cloneSatellite in clonedDataSource.Satellites)
+				// Temporary solution so as to not have to crash the program.
```

Comment:
Temporary, does that mean this should be fixed before merge, or does it function as intended?

Reply:
Temporary meaning that it prevents the program from crashing if it attempts to clone a datasource that has not yet implemented the Clone-method. So in that sense it may not be temporary, meaning that it can remain indefinitely, but when the product is "finished" and all datasources have the clone-method implemented the catch-clause ought never to be triggered.

I'll clarify the comment to make this more explicit.